src/main/kotlin/org/rust/ide/annotator/RsCargoCheckAnnotator.kt

```diff
@@ -79,6 +82,10 @@ class RsCargoCheckAnnotator : ExternalAnnotator<CargoCheckAnnotationInfo, CargoC
     override fun apply(file: PsiFile, annotationResult: CargoCheckAnnotationResult?, holder: AnnotationHolder) {
         if (annotationResult == null) return
 
+        val fileOrigin = (holder.currentAnnotationSession.file as? RsCompositeElement)
```

Comment:
Can we use `file` from function arguments instead of `holder.currentAnnotationSession.file`?